util/crypto/crypto.go

```diff
@@ -0,0 +1,57 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha256"
+	"errors"
+	"io"
+)
+
+func passphraseToKey(passphrase string) []byte {
+	hasher := sha256.New()
+	hasher.Write([]byte(passphrase))
+	key := hasher.Sum(nil)
+	return key
```

Comment:
```suggestion 
	return sha256.Sum256([]byte(passphrase))
```

Could you get away with this? Looks like the hasher might be more designed for when you have chunks of data to add to the hash.

Reply:
Removed hashing since ServeSignature is already 32 random bytes